test/tests/login/LoginViewModelTest.ts

```diff
@@ -309,7 +309,9 @@ o.spec("LoginViewModelTest", () => {
 			verify(credentialsProviderMock.deleteByUserId(testCredentials.userId))
 			verify(credentialRemovalHandler.onCredentialsRemoved(credentialsAndKey))
 			o(viewModel.getSavedCredentials()).deepEquals([])
-			o(viewModel._autoLoginCredentials).equals(null)
+
+			// @ts-ignore
```

Comment:
I think I would rather put `//  visibleForTesting` on the field than ts-ignore in the test.
Especially because this field makes 0 sense and we should refactor the methods to not  have it